pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.isort]
+profile = "black"
+lines_between_types = 0
+lines_after_imports = 1
+combine_as_imports = true
+ignore_whitespace = true
+skip_gitignore = true
```

Comment:
It shows `No newline at end of file` and it looks because of your editor setting. Please make sure it's not shown.